nixos/doc/manual/release-notes/rl-2405.section.md

```diff
@@ -59,6 +59,10 @@ The pre-existing [services.ankisyncd](#opt-services.ankisyncd.enable) has been m
 - `services.avahi.nssmdns` got split into `services.avahi.nssmdns4` and `services.avahi.nssmdns6` which enable the mDNS NSS switch for IPv4 and IPv6 respectively.
   Since most mDNS responders only register IPv4 addresses, most users want to keep the IPv6 support disabled to avoid long timeouts.
 
+- `services.matrix-synapse.sliding-sync` is renamed to `services.matrix-sliding-sync`.
+   neither the nixos module nor the sliding-sync implementation is synapse
```

Comment:
It depends on the synapse database. How is that not Synapse specific?

Reply:
> It depends on the synapse database.

No it doesn't.
The sliding sync proxy uses Client-Server API endpoints and keeps state in a local database.
That database has nothing to do with synapse's database. It's a totally different one.

Note how we create a different database here (and don't re-use synapse's):
https://github.com/NixOS/nixpkgs/blob/dd7a35f7a1af4a1901dbc0df97501bc69b2eab61/nixos/modules/services/matrix/matrix-sliding-sync.nix#L74-L81

Reply:
[from sliding-sync git repo](https://github.com/matrix-org/sliding-sync#usage)
> The proxy works fine with Dendrite and Synapse, but it doesn't work well with Conduit due to spec violations in the state of a room in /sync

I'm running sliding sync successfully with my dendrite homeserver

Reply:
Ah, misremembered some details. Thanks for clearing up.